coursier.bzl

```diff
@@ -278,8 +295,8 @@ def _coursier_fetch_impl(repository_ctx):
         artifact_coordinates.append(utils.artifact_coordinate(a))
         if "exclusions" in a:
             for e in a["exclusions"]:
-                exclusion_lines.append(":".join([a["group"], a["artifact"]]) + \
-                                       "--" + \
+                exclusion_lines.append(":".join([a["group"], a["artifact"]]) +
+                                       "--" +
```

Comment:
Is the removal of the "\" intentional?

Reply:
ugh, didn't realize I included the formatting changes. Yes, this is buildifier's output.

Reply:
There weren't too many, I'm ok merging with the formatting changes :)